### ISSUE TITLE
Duplicate declaration: Package[git] with ensure_packages 

### ISSUE THREAD

**Issue by phifre** [fraenki/puppet-acme#29]
Using `ensure_packages('git')` for puppetmaster setup creates a duplicate declaration error when the package 'git' is included in another module with something else than `ensure_packages('git')` or `ensure => present`. 

A good explanation can be found in this issue for another module:
https://github.com/willdurand/puppet-composer/issues/44

I propose changing the

```puppet
ensure_packages('git')
```

to a 

```puppet
if !defined(Package['git']) {
      ensure_packages('git')
}
```

which will solve the issue.

**Comment by phifre** [comment]
see phifre:issue-29

**Comment by fraenki** [comment]
Fix will be available in the upcoming 2.1.0.
